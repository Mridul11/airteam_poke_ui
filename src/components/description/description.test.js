import { render, screen } from '@testing-library/react';
import MAINIMAGE from '../../assets/images/main.jpeg';
import AVATARIMAGE from '../../assets/images/avatar.jpeg';
import { MemoryRouter } from 'react-router-dom';
import Description from '.';
import AddWatchMedia from '../../utils/test-utils/add-matchmedia';

describe('description component', () => {
  beforeAll(AddWatchMedia);

  const DATA = {
    sprites: {
      other: {
        'official-artwork': {
          front_default: 'https://img.pokemondb.net/artwork/bulbasaur.jpg',
        },
      },
    },
    stats: []
  };

  test('renders description element', () => {
    render(
      <MemoryRouter>
        <Description data={DATA} />
      </MemoryRouter>
    );
    const description = screen.getByText('Pokemon Info');
    expect(description).toMatchSnapshot();
  });
});
