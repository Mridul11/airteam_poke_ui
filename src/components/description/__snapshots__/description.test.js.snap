// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`description component renders description element 1`] = `
<div
  class="ant-descriptions-title"
>
  Pokemon Info
</div>
`;
